README.md

```diff
@@ -1,2 +1,79 @@
-# angelCapstone-project
-This is a work in progress of my capstone project, I will be updating with more information this file. Happy coding :)
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Capstone Project of Module 1.
```

Comment:
[OPTIONAL] It would be nice if you could add a screenshot of your project to your Readme file.

Reply:
Images added to the README file, thank you for your suggestions.